### DISCUSSION TITLE
Create a special type of flipping for opamp symbols

### DISCUSSION THREAD

**Discussion by csrabak** [ra3xdh/qucs_s#999] [Ideas]
In a lot of typical schematics for use of operational amplifiers the drawing could be simplified if the non inverting and inverting inputs can be 'flipped' as in a voltage follower with non inverting input earthed, etc.

In the present implementation of the symbols for opamps due the need of having the power supply pins as well, the GUI "vertical rolling" that flips the component also changes the position of the Vcc and Vss pins.

Could we get a special flipping op for these type of components so the power pins remain in the same place?

Or perhaps have the libraries arranged so an alternate symbol could be selected when placing the circuit element in the schematic?

**Comment by tomhajjar** [comment]
I make custom symbols for devices all the time to "clean up" the schematic. I have many different versions of OpAmp symbols. 

For the 555 timer I have 555_astab, 555_mono, 555_PWM...etc. 

Just make your own subcircuit.

[Symbol_Creation_prj.zip](https://github.com/user-attachments/files/17445408/Symbol_Creation_prj.zip)
